lib/tramway/base_form.rb

```diff
@@ -6,7 +6,7 @@ module Tramway
   class BaseForm
     attr_reader :object
 
-    %i[model_name to_key to_model errors attributes].each do |method_name|
+    %i[id model_name to_key to_model errors attributes].each do |method_name|
```

Comment:
not sure if it really should be default field. There could be models that don't have id at all(like inter tables in has_and_belongs_to_many relationships), or there could be differend name for "id" field

Reply:
About inter tables, I think that Tramway Decorator is not a good option for decorating these objects. By the way, I can't imagine a case for such decorating.

About different names of "id" field: it's interesting, and I know that it's possible. 

What do you think about this solution?

```ruby
    def initialize(object)
      @object = object

      self.class.delegate object.class.primary_key, to: :object
    end
```